jobserver/management/commands/check_rap_api_status.py

```diff
@@ -17,25 +17,23 @@ def handle(self, *args, **options):
             logger.error(exc)
 
     def update_backend_state(backend, request):
-        # Retrieve backend state sent up from job-runner. We might rename the header this is
-        # passed in at some point but for now this is good enough.
-        flags = rap_api.backend_status().get("backends", "")
+        # Retrieve the response gotten from the jobrunner endpoint
+        flags = rap_api.backend_status().get("backends", [])
 
-        # if not flags:  # this line should never be hit because we have a default dict compared with the old way
-        #     return
+        if not flags:
```

Comment:
If we get no backends back from the RAP API, something's probably gone wrong, hasn't it? If you send an invalid client token (i.e. one that doesn't allow access to any backends in the controller), you get a 401, not an empty list. (It's possible we might change that, but even so, an empty list would suggest a problem). Maybe worth at least logging an error before returning?